src/components/NewsletterForm/_newsletter-form.scss

```diff
@@ -6,19 +6,23 @@
 
 .newsletter-form__item {
   margin-bottom: var(--space-md);
-}
 
-.newsletter-form__item--email,
-.newsletter-form__item--consent {
-  @include mq($until: md) {
+  @include mq($until: sm) {
     flex-basis: 100%;
```

Comment:
Does this work in IE11 or do you need to change it to `width: 100%`?

Reply:
I need to change it to `width: 100%`